src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -18,23 +18,71 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
       show: false,
     },
     tooltip: {
-      format: {
-        title: (date) => {
-          return intl.formatDate(date, {
+      contents: function(data) {
```

Comment:
maybe can separate in other file or not? because it's too big the configuration

Reply:
Yes this configuration stands for title, and value in format section for tooltip.

Reply:
Now it reduced its size a lot, perhaps it can be kept here.